userProfile/privateProfile.mjs

```diff
@@ -10,22 +10,23 @@ router.use(
   cors(common_cors)
 )
 
-router.get("/profile", auth0Middleware(), async (req, res) => {
+router.route("/profile").get(auth0Middleware(), async (req, res) => {
   const user = req.user
   if (!user) return respondWithError(res, 401, "Unauthorized user")
 
-  const userObj = new User(user._id)
+  const userObj = new User()
```

Comment:
Just a note to future us that `User` probably deserves the same treatment as `Project` and the normal use would be `new User(_id)` and have a separate way to lookup by `_id` if we only have some properties to query for instead. The constructor would even be able to check for the RERUM prefix and strip to the hex if present so User can be instantiated with agent or id.